parsing/pprintast.ml

```diff
@@ -66,6 +66,8 @@ let fixity_of_string  = function
   | _ -> `Normal
 
 let view_fixity_of_exp = function
+  (* matching on pexp_desc is OK because we're looking for Pexp_ident,
+     which is different from the Pexp_apply node of an extension *)
```

Comment:
I don't love how fiddly these comments are, and how they break the abstraction barrier.  They may be worth it, but I wanted to register my concern.